src/Messages/HubspotMessage.php

```diff
@@ -63,4 +70,17 @@ public function customProperties($customProperties)
 
         return $this;
     }
+
+    /**
+     * Set the message properties of the hubspot message.
+     *
+     * @param array $messageProperties
+     * @return $this
+     */
+    public function messageProperties($messageProperties)
```

Comment:
What's the point of this? If you want to have getter/setter, I'd go with getX / setX / private X; if not, then public X is enough

Reply:
Discussed on DR-1135 PR.